doc/release-notes/0.26/0.26.md

```diff
@@ -50,10 +50,10 @@ The following table covers notable changes in v0.26.0. Further information about
 <tbody>
 
 <tr><td valign="top">
-<a href="https://github.com/eclipse/openj9/issues/7552">#7552</a></td>
-<td valign="top">New default garbage collection (GC) scan ordering mode for the <tt>balanced</tt> policy</td>
+<a href="https://github.com/eclipse/openj9/issues/11278">#11278</a></td>
+<td valign="top">The <strong>jextract</strong> tool is deprecated</td>
 <td valign="top">All versions</td>
-<td valign="top">For performance improvements, GC copy forward operations in the <tt>balanced</tt> policy now use <i>dynamic breadth first scan mode</i>. To revert to the behavior in earlier releases, set <tt>-Xgc:breadthFirstScanOrdering</tt> on the command line when you start your application.</td>
+<td valign="top">The dump extractor tool, <strong>jextract</strong>, is deprecated in this release and will be replaced in a future release. </td>
```

Comment:
We can mention jpackcore here, it's available in the 0.26.0 release.

Reply:
I didn't realise it was already in there. Updated.